pkg/reconciliation/secrets.go

```diff
@@ -142,6 +142,9 @@ func (rc *ReconciliationContext) retrieveSuperuserSecretOrCreateDefault() (*core
 }
 
 func (rc *ReconciliationContext) createInternodeCACredential() (*corev1.Secret, error) {
+	labels := make(map[string]string)
```

Comment:
Instead of creating new empty map, use ``dc.GetDatacenterLabels()``. That adds the necessary clusterLabels also.